### PR TITLE
FEATURE: Fullscreen Secondary inspector

### PR DESCRIPTION
**Pull request by dfeyer**
This change update the CSS to set the secondary inspector in full screen. This change also adapt the ImageEditor and did some cleanup.

NEOS-1796 #resolves

### CODE REVIEW

**Comment by kitsunet** [comment]
looks nice, could it be that your branch is not up to date?

**Comment by dfeyer** [comment]
@kitsunet Yes my master was a bit outdated, just rebased this PR on top of current master

:+1:

File: TYPO3.Media/Resources/Private/Templates/Asset/Edit.html [hunk 1]
```diff
@@ -107,5 +107,5 @@
 </f:section>
 
 <f:section name="Scripts">
-	<script type="text/javascript" src="{f:uri.resource(package: 'TYPO3.Media', path: 'Scripts/edit.js')}"></script>
+	<script type="text/javascript" src="{f:uri.resource(package: 'TYPO3.Neos', path: 'Scripts/edit.js')}"></script>
```
Comment (kdambekalns) [inline TYPO3.Media/Resources/Private/Templates/Asset/Edit.html]: Sorry, but is that good idea? I may not be up to speed when it comes to (de-)coupling Media and Neos, but this looks wrong on many levels…

**Comment by kdambekalns** [comment]
Hm, when cropping an image it looks a little weird: 
other_elements_-_features_-_neos_demo_site

I don't have a good idea on how to make that look less weird, though…

**Comment by aertmann** [comment]
Outcome of https://discuss.neos.io/t/rfc-full-screen-secondary-inspector-by-default/762

I agree with Karsten. It's not always better, depends on screen size and goal.

It would be possible to only do it for smaller screen sizes using media queries. Or a minimum width option that is used to determine if it should be maximized.

**Comment by dfeyer** [comment]
> It would be possible to only do it for smaller screen sizes using media queries. Or a minimum width option that is used to determine if it should be maximized.

I don't like to have different behavior based on screen size. The goal here is to have "focused actions". If you want to crop an image, do it, not reason be disturbed by other possible action (tree, inspector, ...).

Based on @kdambekalns comments, I agree that currently it's a bit akward to have the confirm action and close windows at the opposite of the screen. My suggestion:

1. Move the "close windows action" to the action area, bottom right, with a button and correct label (Cancel)

@aertmann I need your POV ?

**Comment by aertmann** [comment]
> I don't like to have different behavior based on screen size. The goal here is to have "focused actions". If you want to crop an image, do it, not reason be disturbed by other possible action (tree, inspector, ...).
not that simple, in the case of cropping the preview is in the inspector so making it fullscreen degrades the experience..

That's kinda like saying you prefer desktop site on a mobile. I don't see a problem having it behave according to the amount of screen space, that's just natural. The best experience is often not the same depending between the two. What is important is that you have the same possibilities and aren't limited on a small screen.

Moving the actions closer won't really solve the awkwardness if you ask me.

Actually there's another problem here, using the label "Save" doesn't make sense in all cases. During cropping it could work, but should rather be just close or crop. And what about the image selector? Save certainly doesn't make sense there.

**Comment by kdambekalns** [comment]
While this still serves as a nice example of the potential UI issues of such a change, I doubt this will be driven further, since it doesn't even target the React UI. Therefore closing…

**Pull request closed without merging**